### PR TITLE
12.0 [ADD] l10n_be_national_number

### PR DESCRIPTION
**Pull request by victor-champonnois**
Minimal implementation of the Belgian National Number in partner_identification.
The main Roadmap is to add a validation method.

### CODE REVIEW

File: l10n_be_national_number/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import res_partner
+from . import res_partner_id_category
```
Comment (robinkeunen) [inline l10n_be_national_number/models/__init__.py]: I do not see this file in the PR. Did you forget to include it ?

**Review by robinkeunen — CHANGES_REQUESTED** [review]

File: l10n_be_national_number/models/__init__.py (hunk 1 above)
Comment (victor-champonnois) [reply l10n_be_national_number/models/__init__.py]: fixed

**Comment by victor-champonnois** [comment]
@robinkeunen fixed and rebase

**Review by robinkeunen — APPROVED** [review]
LGTM

**Pull request merged**